packages/user-bundle/tests/Unit/UserManipulatorTest.php

```diff
@@ -180,6 +183,9 @@ public function itChangesPassword(): void
                 ->with($user, 'new_password')
                 ->willReturn('hashed_password');
         } elseif (null !== $this->passwordEncoder) {
+            /**
+             * @psalm-suppress UndefinedDocblockClass
```

Comment:
I think this undefinedDocBlockClass comes from the declaration of this property. It should be suppress there if you can.